libraries/AP_RTC/AP_RTC.cpp

```diff
@@ -41,7 +41,7 @@ const AP_Param::GroupInfo AP_RTC::var_info[] = {
 
 void AP_RTC::set_utc_usec(uint64_t time_utc_usec, source_type type)
 {
-    const uint64_t oldest_acceptable_date = 1546300800000; // 2019-01-01 0:00
+    const uint64_t oldest_acceptable_date = 1640995200ULL*1000*1000; // 2022-01-01 0:00 in microseconds
```

Comment:
```suggestion
    const uint64_t oldest_acceptable_date_us = 1640995200ULL*1000*1000; // 2022-01-01 0:00
```

... should've been there from the start :-/

Reply:
Done

Reply:
@Jaaaky sorry, I think you missed the bit about adding `us` to the variable name :-)

I've taken the liberty of making that change here.